openzwavemqtt/models/instance.py

```diff
@@ -82,3 +82,7 @@ def heal_node(self, node_id: int) -> None:
     def cancel_controller_command(self) -> None:
         """Cancel in Controller Commands that are in progress."""
         self.send_command("cancelcontrollercommand")
+
+    def has_node_failed(self, node_id: int) -> None:
+        """Force OZW to test communication with a node."""
+        self.send_command("hasnodefailed", {"node": node_id})
```

Comment:
I'd expect this to return a boolean telling the failed state of the node. Otherwise we should rename the method.